projects/bp-gallery/src/components/views/exhibitions/ExhibitionViewer.tsx

```diff
@@ -66,51 +72,21 @@ const Section = ({ sectionId }: { sectionId: string }) => {
         <div className='flex flex-col gap-4'>
           <div className='text-4xl font-semibold'>{mySection.title}</div>
           <RichText value={mySection.text} />
-          <div className='flex flex-row gap-4 flex-wrap justify-center'>
-            {mySection.dragElements.map((drag, index) => (
-              <div key={index} className='w-max'>
-                <PicturePreview
-                  picture={drag.picture}
-                  height='15rem'
-                  onClick={() => {
-                    setFocusedPicture(drag.picture.id);
-                    pushHistoryWithoutRouter(`/picture/${drag.picture.id}`);
-                  }}
-                />
-              </div>
-            ))}
-          </div>
+          <Gallery
```

Comment:
because I can't set the hight of a PictureGrid and they need to be bigger than a usual PictureGrid. Before I just used Pictures with flex but this didn't worked on safari.